### ISSUE TITLE
Gulp build fails (babel)

### ISSUE THREAD

**Issue by kerimdzhanov** [angular-fullstack/generator-angular-fullstack#2127]
- [x] I understand that GitHub issues are not for tech support, but for questions specific to this generator, bug reports, and feature requests.

Item | Version
----- | -----
generator-angular-fullstack | 4.0.0-rc.0
Node | 6.3.1
npm | 3.10.5
Operating System | OS X 10

Item | Answer
----- | -----
Transpiler | Babel
Markup | HTML
CSS | SCSS
Router | ui-router
Client Tests | Mocha
DB | MongoDB
Auth | Y

Running `$ gulp build` fails with the following error:

```
$ gulp build
[00:27:01] Requiring external module babel-register
(node:15621) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
[00:27:03] Using gulpfile ~/Code/example-project/gulpfile.babel.js
(node:15621) DeprecationWarning: crypto.createCredentials is deprecated. Use tls.createSecureContext instead.
(node:15621) DeprecationWarning: crypto.Credentials is deprecated. Use tls.SecureContext instead.
[00:27:03] Starting 'build'...
[00:27:03] 'build' errored after 945 μs
[00:27:03] Error: Task typings is not configured as a task on gulp.  If this is a submodule, you may need to use require('run-sequence').use(gulp).
    at /Users/master/Code/example-project/node_modules/run-sequence/index.js:20:10
    at Array.forEach (native)
    at verifyTaskSets (/Users/master/Code/example-project/node_modules/run-sequence/index.js:13:11)
    at /Users/master/Code/example-project/node_modules/run-sequence/index.js:32:4
    at Array.forEach (native)
    at verifyTaskSets (/Users/master/Code/example-project/node_modules/run-sequence/index.js:13:11)
    at runSequence (/Users/master/Code/example-project/node_modules/run-sequence/index.js:92:2)
    at Gulp.<anonymous> (/Users/master/Code/example-project/gulpfile.babel.js:401:31)
    at module.exports (/Users/master/Code/example-project/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/master/Code/example-project/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/master/Code/example-project/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/Users/master/Code/example-project/node_modules/orchestrator/index.js:134:8)
    at /usr/local/lib/node_modules/gulp-cli/lib/versioned/^3.7.0/index.js:46:20
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
    at Module.runMain (module.js:577:11)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
```

**Comment by Koslun** [comment]
This is a bug that was fixed with PR #2120. It should be fixed in the next release rc1. Wait for that version or copy the simple change made in the aforementioned PR. I.e. simply remove the typings task from the gulp build task. There essentially should be no typescript or typings mentions.

**Comment by kerimdzhanov** [comment]
Thanks!
